packages/react-router-dev/bin.js

```diff
@@ -5,8 +5,8 @@ let arg = require("arg");
 // default `NODE_ENV` so React loads the proper version in it's CJS entry script.
 // We have to do this before importing `run.ts` since that is what imports
 // `react` (indirectly via `react-router`)
-let args = arg({}, { argv: process.argv.slice(2), stopAtPositional: true });
-if (args._[0] === "dev") {
+let args = arg({}, { argv: process.argv.slice(2), permissive: true });
```

Comment:
`argv` [already defaults to `process.argv.slice(2)`](https://github.com/vercel/arg?tab=readme-ov-file#usage), so we could remove that

```suggestion
let args = arg({}, { permissive: true });
```